binder/views.py

```diff
@@ -65,6 +67,32 @@ def get_default_annotations(model):
 	return annotations
 
 
+def split_path(path):
```

Comment:
Can you add some sort description of what split and join path actually do? And add the type hinting  the arguments? That will make ti much easier to understand what is going on